spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/BinderFactoryConfiguration.java

```diff
@@ -114,6 +117,13 @@ public BinderTypeRegistry binderTypeRegistry(ConfigurableApplicationContext conf
 		return new DefaultBinderTypeRegistry(binderTypes);
 	}
 
+	@Bean
+	@ConditionalOnEnabledHealthIndicator("binders")
+	@ConditionalOnMissingBean(name = "bindersHealthIndicator")
```

Comment:
Is this needed only for the case of `AggregateApplication`? Since we don't `enable` web application for aggregate application, I wonder how health endpoints are used in there.